src/plugins/data/public/plugin.ts

```diff
@@ -154,6 +155,13 @@ export class DataPublicPlugin
       }))
     );
 
+    uiActions.addTriggerAction(
+      'MULTI_VALUE_CLICK_TRIGGER',
```

Comment:
Can we use the same const `MULTI_VALUE_CLICK_TRIGGER` here?

Reply:
If you check the rest of the registers, they dont import from the embeddable plugin due to circular dependencies